Journal/Journal/View Controllers/EntriesTableViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  EntriesTableViewController.swift
+//  Journal
+//
+//  Created by Mitchell Budge on 6/3/19.
+//  Copyright © 2019 Mitchell Budge. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+class EntriesTableViewController: UITableViewController, NSFetchedResultsControllerDelegate {
+    
+    @IBAction func refresh(_ sender: UIRefreshControl) {
+        entryController.fetchEntriesFromServer { (_) in
+            DispatchQueue.main.async {
+                self.refreshControl?.endRefreshing()
+            }
+        }
+    }
+    
+    // MARK: - Table view data source
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return fetchedResultsController.sections?.count ?? 1
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return fetchedResultsController.sections?[section].numberOfObjects ?? 0
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "EntryCell", for: indexPath) as! EntryTableViewCell
+        cell.entry = fetchedResultsController.object(at: indexPath)
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        guard let sectionInfo = fetchedResultsController.sections?[section] else { return nil }
+        return sectionInfo.name
+    }
+    
+    // Override to support editing the table view.
+    override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            let entry = fetchedResultsController.object(at: indexPath)
+            entryController.deleteEntry(entry: entry)
+            tableView.deleteRows(at: [indexPath], with: .fade)
```

Comment:
You don't need this tableview.deleteRows here because its being taken care of by the NSFetchedResultsController